context/app/static/js/components/types.ts

```diff
@@ -89,6 +89,7 @@ export interface Dataset extends Entity {
   dbgap_sra_experiment_url: string;
   files: UnprocessedFile[];
   contributors: ContributorAPIResponse[];
+  contacts?: ContributorAPIResponse[];
```

Comment:
I'm pretty sure we expect `contributors` and `contacts` for all of the entities typed in this file (Datasets, Donors, Samples, Collections, Publications, and Supports). Can you confirm and move these into `Entity` if so?

Reply:
`contributors` and `contacts` do not exist for Donors or Samples.

Reply:
Ah, right. Thanks!

Reply:
You should be able to add these to Collection and Publication.